src/Tizen.Multimedia.MediaPlayer/Player/Player.cs

```diff
@@ -548,6 +548,9 @@ public virtual void Pause()
         ///     -or-<br/>
         ///     It is not able to assign the source to the player.
         ///     </exception>
+        /// <exception cref="PermissionDenied">Thrown when the permission is denied.</exception>
+        /// <exception cref="NotSupportedException">The required feature is not supported.</exception>
```

Comment:
What feature do we need here? Please add `<feature>` for it.

Reply:
The error is not needed. I've removed it.